### DISCUSSION TITLE
I think ALE is not running flake8 w/ appropriate arguments when a file is saved

### DISCUSSION THREAD

**Discussion by AmagicalFishy** [w0rp/ale#4019] [Q&A - Ask for help with problems] [answered]
I added a file, `.config/flake8` with appropriate `max-line-length` arguments to change that to 100 lines. Since, as of October, flake8 no longer supports user-level configuration files, I have `g:ale_python_flake8_options = "--config=$HOME/.config/flake8"` to my `.vimrc`. 

ALE continues to flag lines that are over 79 characters _in addition_ to lines that are over 100 characters. 

- When I open the file, the appropriate lines (over 100 characters) are flagged
- When I save the file, then the lines that are over 79 characters are also flagged
- I have specifically only enabled `flake8` in my `.vimrc`—ensuring that it isn't some other linter that's flagging the lines
- The commands run in ALEInfo appropriately use the `--config` flag
- Running `flake8 --config=$HOME/.config/flake8` manually gives the correct/expected results, so this is isolated to ALE I think
- I've tried `g:ale_linters_explicit = 1` _just in case_ it was still another linter somewhere
- As far as I can tell, there are no other flake8 configuration files, but this shouldn't matter since the `--config` flag should override all other configurations

I think the problem is somewhere in ALE when a file is saved (i.e., everything works correctly when ALE runs after the file is opened, but the configuration isn't considered when the file is saved), but I have no idea where to look. 

Anyone else experience something like this, or have any suggestions for direction?

**Comment by hsanson** [comment, marked as answer]
I am unable to reproduce this issue on my setup. Following exact steps provided I get only lines over 100 properly flagged when opening and when saving a python file.

Can you provide the output of ALEInfo? that may provide more information like specific configurations.  Also version of flake8, vim/neovim.  Finally make sure you are using latest ALE from master that has many fixes.

**Reply by AmagicalFishy** [reply]
... please pretend this post never happened. I looked at the output again in ALEInfo and saw that the error was correct (flagging lines over 100 characters). What confused me so much was that _everything_ looked normal and well. After some searching, I found that I also had Syntastic installed. After deleting Syntastic, everything's working as intended (so the E501 of the 79 lines was Syntastic -__-)

Sorry to waste your time
